src/debugger/Debugger.h

```diff
@@ -407,6 +383,27 @@ class DebuggerRemote : public Debugger {
     Vector<Object*, GCUtil::gc_malloc_allocator<Object*>> m_activeObjects;
 };
 
+class DebuggerC : public Debugger {
```

Comment:
What is the meaning of `DebuggerC`? DebuggerClient?
If so, how about using other name to clearly represent its usage?

Reply:
C api. This class and the DebuggerRemote should be the derived class from Debugger. `DebuggerCAPI`? other suggestion?

On the long run I want to drop these two classes, and the remote debugger should be based on the C API.

Reply:
Oh, I see. `DebuggerC` seems good.